src/meshapi/models/member.py

```diff
@@ -4,8 +4,21 @@
 from django.db.models.fields import EmailField
 from django_jsonform.models.fields import ArrayField as JSONFormArrayField
 
+class ContactPreference(models.TextChoices):
+    EMAIL = 'EMAIL', 'Email'
+    PHONE = 'PHONE', 'Phone'
+    SLACK = 'SLACK', 'Slack'
+    OTHER = 'OTHER', 'Other'
 
 class Member(models.Model):
+
+    contact_preference = models.CharField(
+        max_length=10,
+        choices=ContactPreference.choices,
+        default=ContactPreference.EMAIL,
```

Comment:
I think we should actually default to `None`, so that imported member objects don't get this automatically assigned